### ISSUE TITLE
Doesn't seem necessary to call logging.exception for failed last_executed_query translation

### ISSUE THREAD

**Issue by jacobg** [potatolondon/djangae#519]
When inserting/updating a binary field or any char field with non-ascii characters, the __unicode__ method of InsertCommand will log an exception. It doesn't seem necessary to do so, given it's an expected case. Do you agree?

**Comment by Kazade** [comment]
Yeah this is a bug, the __unicode__ method needs improving generally anyway.
